javascript/sdk/lib/types.ts

```diff
@@ -289,6 +293,7 @@ export interface ProbeActivity {
   endpoint_id: string;
   state: "PROTECTED" | "UNPROTECTED" | "ERROR";
   control: ControlCode;
+  status?: number;
```

Comment:
is this something new you were promised? I dont see this on the BE

Reply:
Yeah I was told that there is going to be a status property on probe activity for use on the hosts table

Reply:
Might not be there yet, but was adding types so that Kenrick can finish the table and then once its in the BE we can remove the optional field part of it